src/main/java/mate/academy/model/Book.java

```diff
@@ -0,0 +1,32 @@
+package mate.academy.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity
+@Data
```

Comment:
Don't use `@Data` for entities https://medium.com/@devchaghtai/understanding-and-resolving-lombok-data-pitfalls-in-jpa-hibernate-entities-ebc76000da18
```suggestion
@Getter
@Setter
@Table(name = ....
```